package.json

```diff
@@ -1,15 +1,8 @@
 {
-<<<<<<< HEAD
   "name": "@elastic/good",
   "description": "Server and process monitoring plugin",
-  "version": "8.1.1-kibana2",
+  "version": "8.2.3-kibana2",
```

Comment:
nit: bump `kibana2` back to `kibana1`?

Reply:
Yeah, took care of that in the next commit